docs/database-classes.md

```diff
@@ -277,8 +277,41 @@ transactor.Transact(db =>
 });
 ```
 
+## Discovery
+
+Starcounter automatically discovers all database classes in the application assembly, which does not include all referenced assemblies. Manual configuration is required to use database classes from referenced assemblies.
+
+### Referenced assembly
+
+```cs
+[Database]
+public abstract class Something
+{
+}
+```
+
+### Application assembly
```

Comment:
```suggestion
### Application assembly
```
And here's the code we need to have in our application for it to detect the database class definition above: